composer.json

```diff
@@ -41,7 +41,7 @@
         "symfony/phpunit-bridge": "~2.7|~3.0|~4.0",
         "twig/twig": "~1.12|~2.0",
         "satooshi/php-coveralls": "^1.0",
-        "phpunit/phpunit": "^6.1"
+        "phpunit/phpunit": "~4"
```

Comment:
`^4.8.36 || ^5.7 || ^6.4` would do the trick and ensure you've got the `PHPUnit\Framework\TestCase` class